libs/common-services/src/main/java/uk/gov/di/ipv/core/library/persistence/DataStore.java

```diff
@@ -72,6 +73,19 @@ public void create(T item) {
         table.putItem(item);
     }
 
+    public void createIfEmpty(T item) {
```

Comment:
Minor, but can we align this with the dynamodb terminology - `createIfNotExists`